Box.V2/Models/BoxRetentionPolicy.cs

```diff
@@ -1,4 +1,4 @@
-﻿using Newtonsoft.Json;
+using Newtonsoft.Json;
```

Comment:
```suggestion
```

Reply:
Looks like this is no longer needed.

Reply:
Sorry, this should have said, "This looks like a Byte Order Mark update on a file with no other changes.  Can you please reset this file to remove it from the PR?"